primitives/consensus/babe/src/digests.rs

```diff
@@ -183,7 +183,26 @@ pub struct NextEpochDescriptor {
 
 /// Information about the next epoch config, if changed. This is broadcast in the first
 /// block of the epoch, and applies using the same rules as `NextEpochDescriptor`.
-pub type NextConfigDescriptor = BabeEpochConfiguration;
+#[derive(Decode, Encode, PartialEq, Eq, Clone, RuntimeDebug)]
+pub enum NextConfigDescriptor {
```

Comment:
This breaks the encoding.

Reply:
This is currently not deployed and even if was it must be unused as there is no way to emit this event currently. I think it's fine that we just break this without migration as the idea of this PR is to make it easier to migrate.